homeassistant/components/sensor/mysensors.py

```diff
@@ -30,13 +50,15 @@
 CONF_PORT = "port"
 CONF_DEBUG = "debug"
 CONF_PERSISTENCE = "persistence"
+CONF_PERSISTENCE_FILE = "persistence_file"
+CONF_VERSION = "version"
 
 ATTR_NODE_ID = "node_id"
 ATTR_CHILD_ID = "child_id"
 
 _LOGGER = logging.getLogger(__name__)
 REQUIREMENTS = ['https://github.com/theolind/pymysensors/archive/'
-                '35b87d880147a34107da0d40cb815d75e6cb4af7.zip'
+                '19413d6eb2f0aefc18b0bc760c21ad1dc981f646.zip'
                 '#pymysensors==0.2']
```

Comment:
If you don't update the version home Assistant won't know to install the new version